libmscore/textbase.cpp

```diff
@@ -328,6 +328,52 @@ bool TextCursor::movePosition(QTextCursor::MoveOperation op, QTextCursor::MoveMo
       return true;
       }
 
+
+
+//---------------------------------------------------------
+//   doubleClickSelect
+//---------------------------------------------------------
+
+void TextCursor::doubleClickSelect()
+      {
+      clearSelection();
+
+      if (!currentCharacter().isSpace()) {
+            //handle double-clicking inside a word
+            int startPosition = _column;
+
+            while (_column > 0 && !currentCharacter().isSpace())
+                  --_column;
+
+            if (currentCharacter().isSpace())
+                  ++_column;
+
+            _selectColumn = _column;
+
+            _column = startPosition;
+            while (_column  < curLine().columns() && !currentCharacter().isSpace())
+                  ++_column;
+            }
+      else {
+            //handle double-clicking between words
+            int startPosition = _column;
+
+            while (_column > 0 && currentCharacter().isSpace())
+                  --_column;
+
+            if (!currentCharacter().isSpace())
+                  ++_column;
+
+            _selectColumn = _column;
+
+            _column = startPosition;
+            while (_column  < curLine().columns() && currentCharacter().isSpace())
+                  ++_column;
+            }
+            updateCursorFormat();
+            _text->score()->addRefresh(_text->canvasBoundingRect());
```

Comment:
Wrong indentation of these two lines: as far as I can see they don't belong to the "if-else" block above so should be indented one level less.

Reply:
OK